atom/browser/api/atom_api_web_contents.cc

```diff
@@ -382,6 +382,26 @@ void WebContents::HandleKeyboardEvent(
     // Escape exits tabbed fullscreen mode.
     ExitFullscreenModeForTab(source);
   } else if (type_ == BROWSER_WINDOW) {
+    // On OSX, if OS event is not existing (because it was forwarded with
+    // `sendInputEvent`) simulate basic operations: undo, redo, cut, copy,
+    // paste, select all.
+    #if defined(OS_MACOSX)
+      if (!event.os_event && (event.modifiers & blink::WebInputEvent::MetaKey)
+        && (event.type == blink::WebInputEvent::RawKeyDown)) {
+        switch (event.windowsKeyCode) {
+          case ui::VKEY_Z:
+            if (event.modifiers & blink::WebInputEvent::ShiftKey) Redo();
+            else
+              Undo();
+          break;
+          case ui::VKEY_X: Cut(); break;
+          case ui::VKEY_C: Copy(); break;
+          case ui::VKEY_V: Paste(); break;
+          case ui::VKEY_A: SelectAll(); break;
```

Comment:
Yes, different `NSWindow`s share the same `NSMenu` in an `NSApplication`, but only the one in focus gets the key shortcut event that was triggered on the `NSMenu`.

The case for me that the chromium window is hidden and an other window is active.

If all the windows would share the same menu at the same time, then if I have two `BrowserWindow` open and I press cmd+a focused on one, everything would be selected in both.